main.go

```diff
@@ -87,6 +87,12 @@ func main() {
 				Value:   nil,
 				Usage:   "trace only the specified event or syscall. use this flag multiple times to choose multiple events",
 			},
+			&cli.StringSliceFlag{
+				Name:    "tag",
```

Comment:
I think tag is not the best name for this feature. maybe "collection" or "events-set" or something like that?

Reply:
OK. I'll go with events-set